tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -729,7 +738,37 @@ Check And Uninstall Operator In Openshift
     END
     Close All Browsers
 
+Check Application Switcher Links To Openshift Cluster Manager
+    [Documentation]    Checks for HTTP status of OCM link in application switcher
+    ${cluster_id}=    Get Cluster ID
+    ${cluster_id}=    Remove String    ${cluster_id}    "
+    ${ocm_staging_link}=    Set Variable    https://qaprodauth.cloud.redhat.com/openshift/details/${cluster_id}
```

Comment:
This line could be inside of the IF statement to understand better the code

Reply:
ack

Reply:
done